README.md

```diff
@@ -2,11 +2,9 @@
 
 This web app is intended for restreaming Ori and the Blind Forest speedrun races.
 
-* Single timer with finish times and animation
-* Change the players and commentary names without reloading the page
-* Skill tracker integration
-* Pulls data from google sheets to show stats/standings
-* Stat fly-ins for main stream page
+* Basic timer
+* Change the players and commentary name without reloading the page
+* Visual Audio selection to show which runner has audio
```

Comment:
I've never updated this file, but if we're going to, we should at least alter line 3 to be more general.